test/gtest/ucs/test_topo.cc

```diff
@@ -46,3 +46,18 @@ UCS_TEST_F(test_topo, get_distance) {
 UCS_TEST_F(test_topo, print_info) {
     ucs_topo_print_info(NULL);
 }
+
+UCS_TEST_F(test_topo, bdf_name) {
+    static const char *bdf_name = "0002:8f:5c.0";
+    ucs_sys_device_t sys_dev    = UCS_SYS_DEVICE_ID_UNKNOWN;
+
+    ucs_status_t status = ucs_topo_find_device_by_bdf_name(bdf_name, &sys_dev);
```

Comment:
maybe also test searching the name without domain?
also some kind of invalid name is worth checking  (that function returns error)